web/onos-gui/src/app/onos-topo/device-detail/device-detail.component.html

```diff
@@ -73,52 +65,59 @@ <h2 class="editable clickable">{{detailsData?.getId()}}</h2>
             <hr>
         </div>
         <div class="middle">
+            <h2 class="changes-title">Attributes</h2>
+            <table>
+                <tbody>
+                    <tr *ngFor="let attribute of detailsData?.getAttributesMap()['map_'] | keyvalue; let i = index">
```

Comment:
I can't see this working out well - if you have 7 attributes you are going to end up with 7 <tr>'s when you only want 4.

I found, something as simple this works:
```
<span *ngFor="let attribute of detailsData?.getAttributesMap()['map_'] | keyvalue">
  <td class="label" width="110"> {{attribute.key}} : </td>
  <td class="value" width="80">{{attribute.value.value}}</td>
</span>
```